src/actions.ts

```diff
@@ -334,5 +334,5 @@ export type Actions<TValue> =
   ;
 
 export function isNgrxFormsAction(action: Action) {
-  return action.type.startsWith('ngrx/forms/');
+  return action.type && action.type.startsWith('ngrx/forms/');
```

Comment:
This PR would be the perfect chance to extract the prefix into a constant and use that everywhere. However, this is optional.